lib/methods/import.js

```diff
@@ -662,27 +666,26 @@ module.exports = self => {
 
       async function update() {
         const _req = req.clone({ mode: doc.aposMode });
-
-        if (doc.aposMode === 'draft') {
-          const existingDoc = await self.apos.doc.db.findOne({ _id: doc._id }, {
-            lastPublishedAt: 1
-          });
-          if (existingDoc) {
-            doc.lastPublishedAt = existingDoc.lastPublishedAt;
```

Comment:
was the `lastPublishedAt` thing important?

Reply:
It's going to get that automatically because we always start with the actual doc as found in the database and apply convert to update properties actually present in the imported doc and the schema, which will leave `lastPublishedAt` unchanged